modules/update-pdb.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/updating/update-cnf-update-prep.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="update-pdb_{context}"]
+= Ensuring that workloads run uninterrupted with pod disruption budgets
+//Please check
+
+Configuration of link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/postinstallation_configuration/post-install-cluster-tasks#post-install-pod-disruption-budgets[PDB (Pod Disruption Budget)] is one of the most important application side settings that is often overlooked. The upgrade of worker nodes can be halted if the PDB is not set properly.  
```

Comment:
Maybe re-write a bit?

= Configuring pod disruption budgets for workloads

Configure pod disruption budgets to <WHAT> <WHY>

(Remove the content about it being really important and overlooked)